.github/workflows/performance_harness_run.yaml

```diff
@@ -12,10 +22,100 @@ defaults:
     shell: bash
 
 jobs:
-  tmp:
-    name: Stub
+
+  v:
+    name: Discover Inputs
+    runs-on: ubuntu-latest
+    outputs:
+      test-params: ${{steps.overrides.outputs.test-params}}
+    steps:
+      - name: Setup Input Params
+        id: overrides
+        run: |
+          echo test-params=testBpOpMode >> $GITHUB_OUTPUT
+
+          if [[ "${{inputs.override-test-params}}" != "" ]]; then
+            echo test-params=${{inputs.override-test-params}} >> $GITHUB_OUTPUT
+          fi
+
+  platforms:
+    name: Run Platforms Workflow
+    uses: ./.github/workflows/platforms.yaml
+    with:
+      override-build-matrix: ${{github.event.inputs.platform-choice}}
+    permissions:
+      packages: write
+      contents: read
+
+  reuse-build:
+    name: Reuse leap build
+    needs: [v]
+    if: |
+      always() &&
+      needs.v.result == 'success'
+    runs-on: ubuntu-latest
+    outputs:
+      build-artifact: ${{steps.downloadBuild.outputs.downloaded-file}}
+    steps:
+        - name: Download builddir
+          id: downloadBuild
+          uses: AntelopeIO/asset-artifact-download-action@v3
+          with:
+            owner: AntelopeIO
+            repo: leap
+            file: build.tar.zst
+            target: ${{github.sha}}
+            artifact-name: ${{github.event.inputs.platform-choice}}-build
+            fail-on-missing-target: false
+            token: ${{github.token}}
```

Comment:
with `v3` you don't need this line, if you want to save a line

Reply:
Done: [d50381b](https://github.com/AntelopeIO/leap/pull/1378/commits/d50381b037be24b1d66b006ab4eb897729613884)